tests/src/ExistingSite/InstallStateTest.php

```diff
@@ -277,39 +277,44 @@ public function testEntityCloneForNewContentType() {
    * Tests tour permission for user roles.
    *
    * - User roles with permission 'access acquia cms tour dashboard'
-   *   should able to Acquia CMS Wizard.
+   *   should be able to access `Acquia CMS Wizard`.
    * - User roles without permission 'access acquia cms tour dashboard'
    *   should not be able to access Acquia CMS Wizard.
    */
   public function testTourPermissions() {
     $assert_session = $this->assertSession();
 
+    // Regular users should not be able to access the dashboard
+    // and tour page.
     $roles = [
       'content_author',
       'content_editor',
-      'content_administrator',
+      // Left empty for anonymous user.
```

Comment:
I don't think this Wizard should be accessible for anonymous user.

Reply:
@chandan-singh7929 Yes, we are testing that only. See [here](https://github.com/acquia/acquia_cms/pull/1385/files#diff-dae9635047e60745eea6143eb30bbad047dd91a1f41a9f7fc3a7559b8ef0f768R302).